src/pybind/mgr/dashboard/frontend/src/app/core/navigation/identity/identity.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { AuthService } from '../../../shared/api/auth.service';
+@Component({
+  selector: 'cd-identity',
+  templateUrl: './identity.component.html',
+  styleUrls: ['./identity.component.scss']
+})
+export class IdentityComponent implements OnInit {
+  username: string;
+  
+  constructor(
+    private authService: AuthService,
+    private router: Router
+  ) { 
+    this.username = localStorage.getItem("dashboard_username");
```

Comment:
You should run `npm run lint` and prettier to align your code style as described [here](https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/HACKING.rst#formating-ts-and-scss-files). After that fix all errors.

Reply:
Yean, I just found that my errors are some linting error, since my first work at community, I made so much error, I will close this merge request, and check code again, commit new code after I modify it.